### DISCUSSION TITLE
How to show variable declared size in pseudo C

### DISCUSSION THREAD

**Discussion by Physicys** [Vector35/binaryninja-api#4826] [Analysis]
How do I show the size of a variable? for example, I set string variable size to 256 but binary ninja decompiler doesn't show it.

![image](https://github.com/Vector35/binaryninja-api/assets/116349837/6eb4a447-7210-465f-aa7f-3a3d50f0f48f)

Both ghidra and ida show the size
![image](https://github.com/Vector35/binaryninja-api/assets/116349837/a414862b-50ee-4e1f-8498-0a7f178e975a)

Source code
```c
#include <stdio.h>

int main() {
  char string[256];

  scanf("%s", &string);
  printf("String: %s\n", string);
}
```

**Comment by plafosse** [comment]
You need to click on `var_108` and hit `y` and then type `char [256]` You can also open up the stack side bar and modify it there as well. We have plans to eventually do this automatically but as for now this step has to be done manually or with a script.
